src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/Results/Net20TwoProjectExpectedRunnerResults.cs

```diff
@@ -111,31 +106,21 @@ private static RunnerResult GetProcessModelInProcessResult(DomainUsage domainUsa
                 case DomainUsage.None:
                     return new RunnerResult
                     {
-                        TestRunner = typeof(AggregatingTestRunner),
-                        SubRunners = new[]
-                        {
-                            new RunnerResult { TestRunner = typeof(LocalTestRunner) },
-                            new RunnerResult { TestRunner = typeof(LocalTestRunner) }
-                        }
```

Comment:
As above - I think this is a break, and should still require an AggregatingTestRunner.

Reply:
Same comment as above as well. :smile: